apps/st2-history/controller.js

```diff
@@ -253,6 +256,46 @@ angular.module('main')
       }
     };
 
+    $scope.rerun = (function () {
+      var rerun = $scope.$new();
+
+      rerun.metaSpec = {
+        type: 'object',
+        properties: {
+          ref: {
+            type: 'string',
+            name: 'Action'
+          }
+        }
+      };
+
+
+      rerun.open = function () {
+        $scope.$root.state.go('^.rerun', {id: $scope.record.id});
+        rerun.payload = _.clone($scope.payload);
```

Comment:
I'm fail to see how it might be considered a feature, sorry =) In case of core.local cmd, the only way i can think of to restore the default value would be to inject something like `pwd`. And I'm still not entirely sure it would work.